packages/react/src/component-tests/IcTooltip/IcTooltip.cy.tsx

```diff
@@ -240,6 +241,20 @@ describe("IcTooltip visual regression and a11y tests", () => {
     });
   });
 
+  it("should render a IcTooltip with a large label", () => {
+    mount(<LargeLabel />);
+
+    cy.checkHydrated(TOOLTIP_SELECTOR);
+    cy.get("button").realHover("mouse");
+
+    cy.checkA11yWithWait();
+    cy.compareSnapshot({
+      name: "Very large label",
+      testThreshold: setThresholdBasedOnEnv(DEFAULT_TEST_THRESHOLD + 0.002),
```

Comment:
I think this threshold might need increasing. If you run the test locally and then view the cypress-image-diff-html-report you'll see what the threshold needs to be set to.